src/entities/dto/list-all-iot-devices-minimal-response.dto.ts

```diff
@@ -35,8 +36,10 @@ export class IoTDeviceMinimalRaw {
 
 export class PayloadDecoderIoDeviceMinimalQuery {
     @ApiPropertyOptional()
-    limit?: number = 20;
+    @IsOptional()
```

Comment:
Is it nessesary to have both ApiPropertyOptional and IsOptional? Sounds like the same functionality.